R/html_document.R

```diff
@@ -292,7 +292,7 @@ html_document <- function(toc = FALSE,
   # pre-processor for arguments that may depend on the name of the
   # the input file AND which need to inject html dependencies
   # (otherwise we could just call the pre_processor)
-  post_knit <- function(metadata, input_file, runtime, ...) {
+  post_knit <- function(metadata, input_file, runtime, encoding, ...) {
```

Comment:
Note that render needs to be updated to pass the encoding (and to check whether post_knit accepts an encoding argument first).